ansible/roles/system_config/templates/sidekiq.j2

```diff
@@ -0,0 +1,34 @@
+[Unit]
+Description=sidekiq
+# start us only once the network and logging subsystems are available,
+# consider adding redis-server.service if Redis is local and systemd-managed.
+After=syslog.target network.target
+
+# See these pages for lots of options:
+# http://0pointer.de/public/systemd-man/systemd.service.html
+# http://0pointer.de/public/systemd-man/systemd.exec.html
+[Service]
+Type=simple
+WorkingDirectory={{ osp_app_directory }}
+# If you use rbenv or rvm:
+# ExecStart=/bin/bash -lc 'bundle exec sidekiq -e production'
+# If you use the system's ruby:
+# ExecStart=bundle exec sidekiq -e production
+ExecStart=/home/decidim/.rbenv/shims/bundle exec sidekiq -e production
+User=decidim
+Group=decidim
+UMask=0002
+
+# if we crash, restart
+RestartSec=1
+Restart=on-failure
+
+# output goes to /var/log/syslog
+StandardOutput=syslog
+StandardError=syslog
```

Comment:
Could you write to `/var/log/sidekiq/production.log` and `/var/log/sidekiq/error.log`